src/components/Autosuggest/Autosuggest.stories.mdx

```diff
@@ -9,16 +9,34 @@ import * as stories from "./Autosuggest.stories.tsx";
 The NYPL Design System does not have its own `Autosuggest` component. Instead,
 we are currently using the
 [react-autosuggest](https://www.npmjs.com/package/react-autosuggest) package.
+We recommend using this package in your application and wrapping it in a custom
+component. The NYPL Design System does have custom styling that targets the
```

Comment:
Could you please add an anchor link to the words "custom component" the drills down to the "Component Implementation" section of the story.